client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -193,6 +193,14 @@ function MetadataDropdownDirective() {
                 _.extend(scope.item, o);
                 scope.change({item: scope.item});
             };
+
+            $timeout(function() {
```

Comment:
Yes, what I observed is, sometimes the variables declared in directive scope doesn't get values until the digest completes.